src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -198,12 +199,17 @@ private VariantOverlapAnnotator initializeOverlapAnnotator(final FeatureInput<Va
         for (final InfoFieldAnnotation annotationType : infoAnnotations) {
             if (annotationType instanceof ReducibleAnnotation) {
                 ReducibleAnnotation currentASannotation = (ReducibleAnnotation) annotationType;
-                if (annotationMap.containsKey(currentASannotation.getRawKeyName())) {
-                    final List<ReducibleAnnotationData<?>> annotationValue = (List<ReducibleAnnotationData<?>>) annotationMap.get(currentASannotation.getRawKeyName());
-                    final Map<String, Object> annotationsFromCurrentType = currentASannotation.combineRawData(allelesList, annotationValue);
-                    combinedAnnotations.putAll(annotationsFromCurrentType);
-                    //remove the combined annotations so that the next method only processes the non-reducible ones
-                    annotationMap.remove(currentASannotation.getRawKeyName());
+                for (final String rawKey : currentASannotation.getRawKeyNames()) {
+                    //here we're assuming that each annotation combines data corresponding to its primary raw key
+                    //AS_QD relies on two keys but doesn't have a combine operation because either GenomicsDB combines the keys
```

Comment:
No, the primary raw key for AS_QUAL is the one you have -- AS_QUALapprox

Having two keys for that one is odd, but I wanted to use a different key when we're approximating rather than when we're using David B.'s full EM.  What do you think about using the same key with an additional flag annotation if we're approximating?

Reply:
Well, if we're allowing multiple raw keys in general, the `AS_QUAL`/`AS_QUALapprox` thing seems reasonable to me. Having to consult a separate annotation to determine how to interpret the AS_QUAL seems worse/more cumbersome.